src/test/java/org/jenkinsci/plugins/workflow/pipelinegraphanalysis/StageTest.java

```diff
@@ -69,20 +69,21 @@ private static void assertChunkBoundary(FlowChunkWithContext chunk, int beforeId
     public void testBlockStage() throws Exception {
         WorkflowJob job = jenkinsRule.jenkins.createProject(WorkflowJob.class, "Blocky job");
 
-        job.setDefinition(new CpsFlowDefinition("" +
-                "node {" +
-                "   stage ('Build') { " +
-                "     echo ('Building'); " +
-                "   } \n" +
-                "   stage ('Test') { " +
-                "     echo ('Testing'); " +
-                "   } \n" +
-                "   stage ('Deploy') { " +
-                "     writeFile file: 'file.txt', text:'content'; " +
-                "     archive(includes: 'file.txt'); " +
-                "     echo ('Deploying'); " +
-                "   } \n" +
-                "}", true));
+        job.setDefinition(new CpsFlowDefinition("""
+                \
+                node {\
+                   stage ('Build') { \
+                     echo ('Building'); \
+                   }\s
+                   stage ('Test') { \
+                     echo ('Testing'); \
+                   }\s
+                   stage ('Deploy') { \
+                     writeFile file: 'file.txt', text:'content'; \
+                     archive(includes: 'file.txt'); \
+                     echo ('Deploying'); \
+                   }\s
```

Comment:
All of these `\`s defeat the purpose of using a multiline constant to begin with, which was to make a legible interpretation of a real program. The `;`s can be dropped if newlines are included.
```suggestion
                node {
                   stage ('Build') {
                     echo ('Building')
                   }
                   stage ('Test') {
                     echo ('Testing')
                   }
                   stage ('Deploy') {
                     writeFile file: 'file.txt', text:'content'
                     archive(includes: 'file.txt')
                     echo ('Deploying')
                   }
```